versioned_docs/version-8.7/self-managed/identity/application-user-group-role-management/manage-groups.md

```diff
@@ -1,21 +1,34 @@
 ---
-id: manage-users-roles-to-group
-title: "Manage users and roles in a group"
-description: "Groups are a way to apply a set of roles and authorizations to users. Use Identity to manage users and roles in a group."
+id: manage-groups
+title: "Manage groups"
+description: "Groups are a way to apply a set of roles and permissions to users. Use Identity to create a group."
 ---
 
-[Users](/self-managed/identity/user-guide/roles/manage-roles.md) and [roles](/self-managed/identity/user-guide/roles/manage-roles.md) can be added or removed from groups.
+Groups are a way to group a set of members. A set of [roles](/self-managed/identity/application-user-group-role-management/manage-roles.md) can be mapped to groups.
 
-:::tip Want to learn more about groups?
-Head over to our documentation on [groups](/self-managed/identity/user-guide/groups/manage-groups.md) to learn more.
-:::
+# Group Management
 
-## Assign members to a group
+## Create a group
 
-:::caution Write access needed
-To assign a user to a group, you must have write access to Identity.
-Read our [guide on managing user access](/self-managed/identity/user-guide/authorizations/managing-user-access.md) to learn more.
-:::
+To create a group, take the following steps:
+
+1. Navigate to the **Groups** tab.
+
+2. Click the **Create group** button located on the top right of the table and a modal will open.
+
+3. Fill in the name of the group and click **Create group**.
+
+On confirmation, the modal closes, the table updates, and your new group is shown.
+
+## Modify or delete a group
+
+1. Navigate to the **Groups** tab.
+
+2. Use the pencil icon to edit the group or the trash icon to remove a group.
```

Comment:
Screenshots would help for this page. Not sure if we want to spend time on this at the moment, so not critical for me.

Reply:
Will not taken action for now. Might revisit later.